mlflow/tracking/client.py

```diff
@@ -269,6 +685,48 @@ def log_batch(self, run_id, metrics=(), params=(), tags=()):
 
         Raises an MlflowException if any errors occur.
         :return: None
+
+        .. code-block:: python
+            :caption: Example
+
+            import time
+
+            from mlflow.tracking import MlflowClient
+            from mlflow.entities import Metric, Param, RunTag
+
+            def print_run_info(r):
+                print("run_id: {}".format(r.info.run_id))
+                print("params: {}".format(r.data.params))
+                print("metrics: {}".format(r.data.metrics))
+                print("tags: {}".format(r.data.tags))
+                print("status: {}".format(r.info.status))
+
+            # Create MLflow entities and a run.
+            timestamp = int(time.time() * 1000)
+            metrics = [Metric('m', 1.5, timestamp, 1)]
+            params = [Param("p", 'p')]
+            tags = [RunTag("t", "t")]
+            client = MlflowClient()
+            run = client.create_run("0")
+
+            # Log entities, terminate the run, and fetch run status
+            try:
+                client.log_batch(run.info.run_id, metrics=metrics, params=params, tags=tags)
+            except MlflowException as ex:
+                print(ex.message)
+            finally:
+                client.set_terminated(run.info.run_id)
+                run = client.get_run(run.info.run_id)
+                print_run_info(run)
```

Comment:
Do we need `try-catch` here?

Reply:
Yes, for the description for [log_batch](https://mlflow.org/docs/latest/python_api/mlflow.tracking.html#mlflow.tracking.MlflowClient.log_batch) function suggests says it can throw an exception.